lib/OpenQA/Git.pm

```diff
@@ -94,4 +97,28 @@ sub commit ($self, $args = undef) {
     return undef;
 }
 
+sub report_status_to_github ($self, $statuses_url, $params, $scheduled_product_id = undef, $callback = undef) {
```

Comment:
Does this method use anything else from the module git? IMHO this would fit better either in the module where used or in a separate module?

Reply:
It only uses the `app` attribute. I just thought it would fit into this module which is basically just a collection of helpers for dealing with Git. I could also create a new `GitProvider` module for that (because it is actually not about running Git itself unlike the other functions here.)

Reply:
yes, GitProvider sounds better. Or actually Github could work as well